atlassian_jwt_auth/contrib/tests/django/test_django.py

```diff
@@ -0,0 +1,85 @@
+import django
```

Comment:
This `django` import shouldn't be here - it should be with the django imports bellow (it isn't a standard module).